_posts/en/newsletters/2025-11-21-newsletter.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Bitcoin Optech Newsletter #381"
+permalink: /en/newsletters/2025/11/21/
+name: 2025-11-21-newsletter
+slug: 2025-11-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+- **Modeling stale rates by propagation delay and mining centralization:**
+  Antoine Poinsot [posted to
+  delving][antoine delving] about modeling stale block rates and how
+  block propagation time affects a miner's revenue as a function of its
+  hashrate. He set up a base case scenario where all miners act
+  rationally, meaning that if they received a new block, then they
+  would immediately start mining on top of it and publish immediately.
+  This would lead to a revenue that is proportional to their share of
+  the hashrate.
+
+  In their model with uniform block propagation, they had two
```

Comment:
Not clear who the they and we is here? Perhaps we can word differently

Reply:
Feel free to use the less ambiguous "his" for me, if that's me that you were referencing with "their"